src/sentry/static/sentry/app/views/organizationDashboard/deploys.jsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'react-emotion';
+import {Flex, Box} from 'grid-emotion';
+import moment from 'moment-timezone';
+
+import SentryTypes from 'app/proptypes';
+import Button from 'app/components/buttons/button';
+import Link from 'app/components/link';
+import {t} from 'app/locale';
+import TextOverflow from 'app/components/textOverflow';
+
+const DEPLOY_COUNT = 2;
+
+export default class Deploys extends React.Component {
+  static propTypes = {
+    project: SentryTypes.Project,
+    orgId: PropTypes.string,
+  };
+
+  render() {
+    const {project, orgId} = this.props;
+
+    const projectId = project.slug;
+
+    const deploys = (project.latestDeploys || [])
+      .sort((a, b) => new Date(b.dateFinished) - new Date(a.dateFinished))
+      .slice(0, DEPLOY_COUNT);
+
+    if (deploys.length) {
+      return (
+        <DeployBox p={2} pb={0}>
+          <Heading>{t('Latest deploys')}</Heading>
+          <div>
+            {deploys.map(deploy => (
+              <Deploy
+                key={deploy.version}
+                deploy={deploy}
+                projectId={projectId}
+                orgId={orgId}
+              />
+            ))}
+          </div>
+        </DeployBox>
+      );
+    } else {
```

Comment:
This `else` is un-needed.

Just a style opinion, I think it's easier to read if the "main" component is the "default" condition and placed at the end. e.g.

```
if (cond) return <NoDeploys />;

return <MainStuff />;
```

Thoughts?

Reply:
Yeah, I've seen this style often in our codebase. I like symmetric if/else better though.